jdaviz/components/data_menu_add_data.vue

```diff
@@ -0,0 +1,101 @@
+<template>
+  <v-menu
+    absolute
+    offset-y
+    left
+    v-if="dataset_items.length > 0 || subset_tools.length > 0"
+    >
+    <template v-slot:activator="{ on, attrs }">
+      <j-tooltip
+        v-if="dataset_items.length > 0 || subset_tools.length > 0"
+        tooltipcontent="Add data or subset to viewer"
+      >
+        <v-btn
+          icon
+          v-bind="attrs"
+          v-on="on"
+        >
+          <v-icon>mdi-plus</v-icon>
+        </v-btn>
+      </j-tooltip>
+    </template>
+    <v-list dense style="width: 200px; max-height: 300px; overflow-y: auto;">
+      <v-subheader v-if="dataset_items.length > 0" class="strike"><span>Load Data</span></v-subheader>
+      <v-list-item
+        v-for="data in dataset_items"
+      >
+        <v-list-item-content>
+          <j-tooltip tooltipcontent="add data to viewer">
+            <span
+              style="cursor: pointer; width: 100%"
+              @click="() => {$emit('add-data', data.label)}"
+            >
+              {{ data.label }}
+            </span>
+          </j-tooltip>
+        </v-list-item-content>
+      </v-list-item>
+      <v-subheader v-if="subset_tools.length > 0" class="strike"><span>Create Subset</span></v-subheader>
+      <v-list-item
+        v-if="subset_tools.length > 0"
+      >
+        <v-list-item-content style="display: inline-block">
+          <j-tooltip
+            v-for="tool in subset_tools"
+            span_style="display: inline-block"
+            :tooltipcontent="'Create new '+tool.name+' subset'"
+          >
+            <v-btn 
+              icon
+              @click="() => {$emit('create-subset', tool.name)}"
+            >
+              <img :src="tool.img" width="20"/>
+            </v-btn>
+          </j-tooltip>
+        </v-list-item-content>
+      </v-list-item>
+    </v-list>
+  </v-menu>
+
+</template>
+
+<script>
+module.exports = {
+  props: ['dataset_items', 'subset_tools'],
+};
+</script>
+
+<style scoped>
+.strike {
+    display: block;
+    text-align: center !important;
+    height: 28px !important;
+}
+
+.strike > span {
+    position: relative;
+    display: inline-block;
+    text-transform: uppercase;
+    font-weight: bold;
+}
+
+.strike > span:before,
+.strike > span:after {
+    content: "";
+    position: absolute;
+    top: 50%;
+    width: 9999px;
+    height: 1px;
+    background: gray;
+}
+
+.strike > span:before {
+    right: 100%;
+    margin-right: 6px;
+}
+
+.strike > span:after {
+    left: 100%;
+    margin-left: 6px;
+}
```

Comment:
reviewers: do you see the horizontal lines on these headers in the menu?  I was having some problems with them but it could be a browser/jupyter css caching issue that makes it hard to tell.  I can also cross-reference with another browser - if they aren't working we should just remove them and try to re-implement during the general UI tweaking of the entire data-menu, I don't think its worth chasing down now for something that is just UI sugar.

Reply:
Is this what you'd expect to see?
![image](https://github.com/user-attachments/assets/d79a0157-1466-4b98-8495-a4d965a72790)

Reply:
in the "+" submenu

Reply:
(also see that we have a dark mode issue, the icons shouldn't invert color back to black... I'll try to fix that)

Reply:
![image](https://github.com/user-attachments/assets/e96d1aee-a2b3-48f2-b5eb-a1854e46f3b8)

Reply:
ok, no horizontal lines or text styling.  I'll remove for now then and we can revisit during the final UI tweaks